Makefile

```diff
@@ -193,6 +194,9 @@ $(filter-out $(objdir)/uftrace.o,$(UFTRACE_OBJS)): $(objdir)/%.o: $(srcdir)/%.c
 $(objdir)/version.h: PHONY
 	@$(srcdir)/misc/version.sh $@ $(VERSION_GIT)
 
+$(objdir)/autoargs.h: PHONY
+	@$(objdir)/utils/gen-autoargs.py $(objdir)/utils/prototypes.h
```

Comment:
I think we don't need to run `gen-autoargs.py` always.  If we provide autoargs.h, it will run only after prototype.h was changed.  Also I'd like put the script and prototype header in the "misc" directory rather than "utils".

Reply:
Hi, I've just updated as you said. I removed the commit for `Makefile` and added `autoargs.h` instead. Also I moved those files to `misc`. Thanks.